azplugins/BounceBackNVEGPU.cu

```diff
@@ -10,7 +10,10 @@
  */
 
 #include "BounceBackNVEGPU.cuh"
-#include "BounceBackGeometry.h"
```

Comment:
I'm in favor of leaving the mpcd headers in `BounceBackGeometry.h` and not explicitly including them individually here, so you can revert this change. Otherwise, remove the headers from `BounceBackGeometry.h`.

Reply:
I've put the headers in `BounceBackGeometry.h` and removed them from  `BounceBackNVEGPU.cu` and module.cc .